sa/sa.go

```diff
@@ -400,38 +409,39 @@ func (ssa *SQLStorageAuthority) GetCertificate(ctx context.Context, serial strin
 		return core.Certificate{}, err
 	}
 
-	certObj, err := ssa.dbMap.Get(core.Certificate{}, serial)
-	if err != nil {
-		return core.Certificate{}, err
-	}
-	if certObj == nil {
-		ssa.log.Debug(fmt.Sprintf("Nil cert for %s", serial))
+	var cert core.Certificate
+	err := ssa.dbMap.SelectOne(&cert, fmt.Sprintf("SELECT %s FROM certificates WHERE serial = %q", CertificateFields, serial))
```

Comment:
Definitely let's not DIY our SQL interpolation like this. Use Gorp's interpolation technique.